nullaway/src/main/java/com/uber/nullaway/handlers/BaseNoOpHandler.java

```diff
@@ -199,8 +201,9 @@ public Optional<ErrorMessage> onExpressionDereference(
   }
 
   @Override
-  public boolean includeApInfoInSavedContext(AccessPath accessPath, VisitorState state) {
-    return false;
+  public Predicate<AccessPath> getAccessPathPredicateForNestedMethod(
+      TreePath path, VisitorState state) {
+    return CompositeHandler.FALSE_AP_PREDICATE;
```

Comment:
Why is this a field of `CompositeHandler`, rather than say `Handler` (where we define e.g. `NullnessHint`)

Reply:
I guess it's not easy to have it in `Handler` and package visible... one other option is something like `AccessPathPredicates.java` in this same package, unless you feel this is very much linked to `CompositeHandler` in terms of its usage. Either way, I don't feel super strongly about it, if you want to leave it here it might be fine too.

Reply:
You're right, creating an `AccessPathPredicates` class is better, I'll do that.